hal/targets.json

```diff
@@ -1939,7 +1939,8 @@
             "NRF52_PAN_63"
         ],
         "device_has": ["ANALOGIN", "ERROR_PATTERN", "I2C", "I2C_ASYNCH", "INTERRUPTIN", "LOWPOWERTIMER", "PORTIN", "PORTINOUT", "PORTOUT", "PWMOUT", "RTC", "SERIAL", "SERIAL_ASYNCH", "SLEEP", "SPI", "SPI_ASYNCH", "SPISLAVE"],
-        "release_versions": ["2", "5"]
+        "release_versions": ["2", "5"],
+        "public": true
```

Comment:
I was asked by @screamerbg to do so:

The steps to follow by the end of this week:
•	Pull request with “public” flag set for the NRF52_DK target in targets.json (https://github.com/mbedmicro/mbed repository)
Could you took to him in this topic?